src/Raven.Server/Documents/Sharding/Handlers/Processors/Replication/ShardedReplicationHandlerProcessorForGetPerformance.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections.Generic;
+using JetBrains.Annotations;
+using Raven.Server.Documents.Handlers.Processors.Replication;
+using Raven.Server.Documents.Replication.Incoming;
+using Raven.Server.Documents.Replication.Outgoing;
+using Raven.Server.ServerWide.Context;
+using Sparrow.Utils;
+
+namespace Raven.Server.Documents.Sharding.Handlers.Processors.Replication
+{
+    internal class ShardedReplicationHandlerProcessorForGetPerformance : AbstractReplicationHandlerProcessorForGetPerformance<ShardedDatabaseRequestHandler, TransactionOperationContext>
```

Comment:
I think we need here 2 handlers, one for proxy and the other one is to get details from the orchestrator node.

For now let's go only with the proxy and handle the orchestrator  at later point

Reply:
Done